src/bidmanager.js

```diff
@@ -273,19 +278,26 @@ exports.executeCallback = function (timedOut) {
   }
 };
 
+exports.externalCallbackReset = function () {
+  externalCallbacks.all.called = false;
```

Comment:
`externalCallbacks` has evolved into complex object. Should we formalize this into int's own file / class constructor?